lib/jar.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import * as crypto from 'crypto';
+import * as fs from 'fs';
+
+export function getTempPomPathForJar(root: string, jar: string): string {
+  const sha1 = getSha1ForJar(path.join(root, jar));
+
+  // return path to root of pom.xml
+  return '';
+}
+
+export function isJar(file: string) {
+  return file.match(/\.(([jw]ar)|(zip))$/);
```

Comment:
do we support enterprise achieves too? (ear)

Reply:
If they are on maven central, don't see why not

Reply:
will add when there is customer demand